app-crypt/xmrig/xmrig-20251105.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="CPU and GPU miner"
+DESCRIPTION="RandomX, KawPow, CryptoNight and GhostRider unified CPU/GPU miner and RandomX benchmark"
+HOMEPAGE="https://github.com/xmrig/xmrig"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="4cdc35f96648f8363d26526d1bfa90502833bac9"
+SOURCE_URI="https://github.com/xmrig/xmrig/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="b80a166521c7d447775e5c012b36ef3898388d8f8f72bf4e5fbe9769de94a60a"
+ARCHITECTURES="all ?x86 ?x86_gcc2"
```

Comment:
There is no primary architecture "x86", and this won't build with gcc2 (after a quick look at the code, this seems to use modern C++).

```suggestion
ARCHITECTURES="all !x86_gcc2"
```

If you want to add x86, you have to add `SECONDARY_ARCHITECTURES="x86"` and adapt the recipe to use `$secondaryArchSuffix`.